### ISSUE TITLE
image related to tezos utils cannot be found

### ISSUE THREAD

**Issue by Michshelle** [oxheadalpha/tezos-k8s#580]
Hello,
I tried to spin up a node today but the below problem cannot be solved by myself.  Is it a bug or can I fix it?  The weird thing is it seems the image for tezos node is okay but only the util image failed.

My os is raspberry os; I'm deploying k8s on physical servers.  Let me know if you need more details.  Thank you.

`Normal   Pulled     5m37s                  kubelet            Successfully pulled image "tezos/tezos:v16.1" in 1m48.418331503s (1m48.418393354s including waiting)
  Normal   Created    5m36s                  kubelet            Created container config-init
  Normal   Started    5m36s                  kubelet            Started container config-init
  Warning  Failed     4m23s (x3 over 5m12s)  kubelet            Failed to pull image "ghcr.io/oxheadalpha/tezos-k8s-utils:6.19.1": rpc error: code = NotFound desc = failed to pull and unpack image "ghcr.io/oxheadalpha/tezos-k8s-utils:6.19.1": no match for platform in manifest: not found`

EDIT: 
`helm upgrade tezos-mainnet oxheadalpha/tezos-chain -n oxheadalpha --set nodeSelector.hostname=rsblue` 
Basically I'm following doc published on https://tezos-k8s.xyz/.  
A little bit diversion is I created local pvc under namespace oxheadalpha before the step.

**Comment by harryttd** [comment]
Hi :)

Your platform may not be compatible with the image. Can you check? 

uname -m && uname -s

Shelling into a utils container on our eks nodes:
```
/ # uname -s && uname -m
Linux
x86_64
```

Notice there is a tab for an image for example, for linux/amd64. https://github.com/oxheadalpha/tezos-k8s/pkgs/container/tezos-k8s-utils/84965234?tag=6.19.1

**Comment by Michshelle** [comment]
> https://github.com/oxheadalpha/tezos-k8s/pkgs/container/tezos-k8s-utils/84965234?tag=6.19.1

Thank you, Harry.  I think it gets a bit closer.  My platform is aarch64, so I think it should be unknown/unknown.  So I decided to ctr image pull it first and then load it in k8s.  However, a new issue popped out.  Below was what I did.
`sudo ctr image pull ghcr.io/oxheadalpha/tezos-k8s-utils:6.19.1@sha256:9f9f598d08f1b8f338c6ba9f41af61034dc646657e6577523b5064cee436ab14`
At this point, I've seen an error log but I ignored it and continued the processes.
error="failed to extract layer sha256:4c529f99308fff480d53c7848ffa24e0190102b5e96e050672554e722a81f398: 
failed to get stream processor for application/vnd.in-toto+json: no processor for media-type: unknown" key="extract-371863934-HdmA sha256:4c529f99308fff480d53c7848ffa24e0190102b5e96e050672554e722a81f398"

`sudo ctr image export tezos-k8s-utils:6.19.1.tar ghcr.io/oxheadalpha/tezos-k8s-utils:6.19.1@sha256:9f9f598d08f1b8f338c6ba9f41af61034dc646657e6577523b5064cee436ab14 --platform unknown/unknown`
Got no error message here.

`sudo ctr -n=k8s.io images import tezos-k8s-utils:6.19.1.tar`
And the error message this time,
INFO[0000] apply failure, attempting cleanup   
error="failed to extract layer sha256:4c529f99308fff480d53c7848ffa24e0190102b5e96e050672554e722a81f398: 
failed to get stream processor for application/vnd.in-toto+json: 
no processor for media-type: unknown" key="extract-459270993-_v1f sha256:4c529f99308fff480d53c7848ffa24e0190102b5e96e050672554e722a81f398"

Concerning the error messages, I think they are talking about the same thing.  However due to limited docker related skills, I have no idea how to solve this problem.  I think another way to make it work is unzip the latest release and "kubectl create" those yaml files one-by-one?  But I suppose that is even more complicated?

**Comment by Michshelle** [comment]
By editing images listed in statefulset.app, the mentioned problem seem be solved.
`kubectl edit StatefulSet/rolling-node -n oxheadalpha`
I found an updated image with a new tag, https://github.com/oxheadalpha/tezos-k8s/pkgs/container/tezos-k8s-utils/89321958?tag=user-configurable-s3-bucket, thus basically I replaced the previous "6.19.1" with "user-configurable-s3-bucket". 
 Although some other new issues emerged, it is no longer a image-pulling problem.
